easybuild/easyconfigs/n/Nextflow/Nextflow-22.10.1.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'Binary'
+
+name = 'Nextflow'
+version = '22.10.1'
+
+homepage = 'https://www.%(namelower)s.io/'
+description = """Nextflow is a reactive workflow framework and a programming DSL 
+ that eases writing computational pipelines with complex data"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/%(namelower)s-io/%(namelower)s/releases/download/v%(version)s/']
+sources = ['%(namelower)s-%(version)s-all']
```

Comment:
@jpecar Do you know what the difference is between using `nextflow-22.10.1-all`  like you're doing here, and installing using `v22.10.1.tar.gz` as is done in existing `Nextflow` easyconfigs?

Reply:
> @jpecar Do you know what the difference is between using `nextflow-22.10.1-all` like you're doing here, and installing using `v22.10.1.tar.gz` as is done in existing `Nextflow` easyconfigs?

nextflow-all is the full binary, at some point this was switched to only a shell script that downloads it from somewhere on each invocation of nextflow. To me that doesn't look like in the spirit of EESSI so I intentionally picked up the older style .eb that installs the binary.

Reply:
OK, thanks for clarifying!